apps/website/src/utils/sw.ts

```diff
@@ -12,3 +12,13 @@ export default function registerServiceWorker(path: string, scope = "/") {
       });
   }
 }
+
+export function unregisterServiceWorker(path: string, scope = "/") {
+  if (typeof navigator !== "undefined" && "serviceWorker" in navigator) {
+    navigator.serviceWorker.getRegistration(scope).then((registration) => {
+      if (registration) {
```

Comment:
```suggestion
      if (registration?.active && new URL(registration.active.scriptURL).pathname === path) {
```
If we want to enforce the path?

Reply:
I think just using the scope is enough?! Ij ust forgot to remove the path param while switching between the branches.